Clicker/ViewControllers/PollBuilderViewController.swift

```diff
@@ -303,11 +326,16 @@ class PollBuilderViewController: UIViewController, QuestionDelegate, PollBuilder
     func updateQuestionType() {
         questionType = questionType.other
         updateQuestionTypeButton()
-        if questionType == .multipleChoice {
+        guard let type = questionType else {
+            print("question type must be initalized before updateQuestionTypeCalled.")
+            return
+        }
+        switch type {
```

Comment:
instead of having this switch you can do:
`mcPollBuilder.isHidden = type == .freeResponse`
`frPollBuilder.isHidden = type == .multipleChoice`